assets/components/featuredDigitalPack/flashSaleDigitalPack.scss

```diff
@@ -31,6 +31,7 @@
   .component-flash-sale-featured-digital-pack__countdownbox {
     border-left: 1px solid gu-colour(garnett-neutral-4);
     padding-left: 5px;
+    padding-bottom: 70px;
```

Comment:
Can you make this `120px`? It's clipping the CTA on the narrower breakpoints.

Reply:
done

Reply:
i thought the clipping was intentional however? provided the image goes under the button